tools/src/test/java/com/nedap/archie/aom/CComplexObjectProxyTest.java

```diff
@@ -0,0 +1,15 @@
+package com.nedap.archie.aom;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CComplexObjectProxyTest {
+
+    @Test
+    public void isLeaf() {
+        CComplexObject cComplexObject = new CComplexObject();
+
+        assertTrue(cComplexObject.isLeaf());
```

Comment:
In operational templates, I think these can contain child nodes. Same with Archetype slots. I'll check later, and fix both implementation and test. Unless Jelte wants to do this? :)

Reply:
> Unless Jelte wants to do this? :)
No, not now.

Reply:
I checked this, and they can definately not contain children. So this test plus implementation is correct. Archetype roots actually can, but those are CComplexObjects and not tested separately. So I'm merging this PR :)